distributed/core.py

```diff
@@ -406,7 +406,7 @@ async def listen(self, port_or_addr=None, allow_offload=True, **kwargs):
         )
         self.listeners.append(listener)
 
-    async def handle_comm(self, comm, shutting_down=shutting_down):
```

Comment:
Grepping through the codebase I didn't see us ever utilize this `shutting_down=` keyword, so I've opted to inline `sys.is_finalizing` instead